arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt

```diff
@@ -191,49 +78,6 @@ class ScheduleTest : ArrowFxSpec() {
       sideEffect.counter shouldBe 1
     }
 
-    "Schedule.spaced()" {
-      forAll(Gen.intSmall().filter { it > 0 }, Gen.intSmall().filter { it > 0 }.filter { it < 1000 }) { i, n ->
-        val res = Schedule.spaced<ForId, Any>(Id.monad(), i.seconds).runIdSchedule(0, n)
-
-        res.forAll { it.delay.nanoseconds == i.seconds.nanoseconds && it.cont }
-      }
-    }
-
-    "Schedule.fibonacci()" {
-      forFew(5, Gen.intSmall().filter { it > 0 }.filter { it < 10 }, Gen.intSmall().filter { it > 0 }.filter { it < 10 }) { i, n ->
-        val res = Schedule.fibonacci<ForId, Any?>(Id.monad(), i.seconds).runIdSchedule(0, n)
-
-        val sum = res.fold(0L) { acc, v ->
-          acc + v.delay.amount
-        }
-        val fibSum = fibs(i.toLong()).drop(1).take(res.size).sum()
-
-        res.forAll { it.cont } && sum == fibSum
-      }
-    }
-
-    "Schedule.linear()" {
```

Comment:
Are those to be replaced, with something else, or is this not required anymore? 
cc @nomisRev

Reply:
Arrow Fx is deprecated and will be removed in 0.13.0 so this doesn't need to be replaced.

Reply:
thanks I was asking the same myself 👍